beets/importer.py

```diff
@@ -1019,7 +1019,7 @@ def find_duplicates(self, lib):
         # temporary `Item` object to generate any computed fields.
         tmp_item = library.Item(lib, **info)
         keys = config["import"]["duplicate_keys"]["item"].as_str_seq()
-        dup_query = library.Album.all_fields_query(
+        dup_query = library.Item.match_all_query(
```

Comment:
It seemed to me this was supposed to be `Item` instead of `Album`?

Reply:
I don't think it's the same, as it uses `Album._fields` and `Item._fields` which are different, see

```python
    @classmethod
    def field_query(
        cls,
        field,
        pattern,
        query_cls: Type[FieldQuery] = MatchQuery,
    ) -> FieldQuery:
        """Get a `FieldQuery` for this model."""
        return query_cls(field, pattern, field in cls._fields)

    @classmethod
    def all_fields_query(
        cls: Type["Model"],
        pats: Mapping,
        query_cls: Type[FieldQuery] = MatchQuery,
    ):
        """Get a query that matches many fields with different patterns.

        `pats` should be a mapping from field names to patterns. The
        resulting query is a conjunction ("and") of per-field queries
        for all of these field/pattern pairs.
        """
        subqueries = [cls.field_query(k, v, query_cls) for k, v in pats.items()]
        return AndQuery(subqueries)
```